eclair-core/src/main/scala/fr/acinq/eclair/blockchain/WatcherTypes.scala

```diff
@@ -26,20 +26,28 @@ import scodec.bits.ByteVector
 import scala.util.{Failure, Success, Try}
 
 /**
-  * Created by PM on 19/01/2016.
-  */
+ * Created by PM on 19/01/2016.
+ */
 
 // @formatter:off
 
 sealed trait Watch {
   def channel: ActorRef
```

Comment:
`replyTo` is all the rage these days and it's actually a better name since for `WatchSpentBasic` it's the router.
```suggestion
  def replyTo: ActorRef
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1500/commits/7f319935c023e03fa83e2dc5c0e702d26f63c212